app/src/main/java/com/sun/wsm_android_v2/data/model/DefautGroups.kt

```diff
@@ -0,0 +1,14 @@
+package com.sun.wsm_android_v2.data.model
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+class DefautGroups {
+    @SerializedName(GROUP)
```

Comment:
không cần phải tạo biến hằng đâu e. đối với key json thì không cần
```suggestion
    @SerializedName("1")
```